ProcessManager.py

```diff
@@ -6,14 +6,24 @@
 from multiprocessing.context import Process # Processes are fully functional on Windows (as of August 20, 2017).
 import time
 import sys
+from subprocess import Popen
+
+class RoboProcesses:
+            def _init_(self, cmd):
```

Comment:
Should be two underscores for the constructor method (`__init__`). 😃
Also, super nitpicky thing, but the block starting on line 12 should only be intented 4 spaces in keeping with [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation).